assemblies/karaf/src/main/descriptors/standalone-assembly.xml

```diff
@@ -27,17 +27,25 @@
 
     <!-- Expanded Karaf Standard Distribution -->
     <fileSet>
-      <directory>target/dependencies/apache-karaf-${karaf.version}</directory>
+      <directory>${project.build.directory}/dependencies/apache-karaf-${karaf.version}</directory>
       <outputDirectory>/apache-karaf-${karaf.version}</outputDirectory>
       <excludes>
         <exclude>etc/org.apache.karaf.features.cfg</exclude>
         <exclude>etc/startup.properties</exclude>
         <exclude>etc/custom.properties</exclude>
+        
+        <exclude>**/demos/**</exclude>
+        <exclude>LICENSE</exclude>
```

Comment:
We do not need them and the excluding is being done in pentaho-karaf-assembly project.